milo/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@adobecom/milo",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Website foundation technology.",
+  "scripts": {
+    "test": "wtr --config ./web-test-runner.config.mjs \"./test/**/*.test.(js|html)\" --node-resolve --port=2000 --coverage --concurrent-browsers 4",
+    "test:watch": "npm test -- --watch",
+    "test:file": "wtr --config ./web-test-runner.config.mjs --node-resolve --port=2000 --coverage",
+    "libs": "hlx up --port=6456",
+    "lint": "npm run lint:js && npm run lint:css",
+    "lint:js": "eslint .",
+    "lint:css": "stylelint 'blocks/**/*.css' 'styles/*.css'",
+    "build:htm-preact": "microbundle ./build/htm-preact.js -o ./libs/deps/htm-preact.js -f modern --no-sourcemap --target web; mv ./libs/deps/htm-preact.modern.mjs ./libs/deps/htm-preact.js",
+    "build:preact-debug": "microbundle ./build/htm-preact-debug.js -o ./libs/deps/htm-preact-debug.js -f modern --no-sourcemap --target web; mv ./libs/deps/htm-preact-debug.modern.mjs ./libs/deps/htm-preact-debug.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/adobecom/milo.git"
+  },
+  "author": "Adobe",
+  "license": "Apache License 2.0",
+  "bugs": {
+    "url": "https://github.com/adobecom/milo/issues"
+  },
+  "homepage": "https://github.com/adobecom/milo#readme",
+  "dependencies": {
+    "@preact/signals": "1.0.4",
+    "htm": "^3.1.1",
+    "preact": "^10.11.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-dom-components": "3.0.0"
+  },
+  "devDependencies": {
+    "@babel/core": "7.17.7",
+    "@dexter/offer-selector-tool": "1.7.2",
```

Comment:
I was thinking OST loaded inside an iframe similar to library with a loose coupling.
OST is a authoring only tool and should have 0 impact on the pure frontend stack.

the existing merch block is already doing a good job for rendering the prices and CTAs.

Reply:
Removed since it's not needed. ✅